File: app/src/main/java/me/gilo/starter/db/NoteDatabase.java
package me.gilo.starter.db;


import androidx.room.Database;
import androidx.room.RoomDatabase;
import me.gilo.starter.dao.DaoAccess;
import me.gilo.starter.models.Note;

@Database(entities = {Note.class}, version = 1, exportSchema = false)
public abstract class NoteDatabase extends RoomDatabase {

    public abstract DaoAccess daoAccess();
}

File: localdataprovider/src/main/java/me/gilo/localdataprovider/db/NoteDatabase.kt
package me.gilo.localdataprovider.db


import androidx.room.Database
import androidx.room.RoomDatabase
import me.gilo.localdataprovider.dao.DaoAccess
import me.gilo.localdataprovider.models.NoteEntity


@Database(entities = [NoteEntity::class], version = 1, exportSchema = false)
abstract class NoteDatabase : RoomDatabase() {

    abstract fun daoAccess(): DaoAccess
}
